### PR TITLE
Support the new dashboard widgets

### PR DESCRIPTION
**Pull request by mfalesni**
{{pytest: cfme/tests/intelligence/test_dashboard.py -v -k 'operation or widget' --long-running }}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit f9828d447164f2f37fb554d21a067cd71e1dc85d:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit e89e4ee809be53992699dc83969df90681304581:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/dashboard.py [hunk 1]
```diff
@@ -112,36 +137,66 @@ def time_next(self):
...
+        else:
+            self.open_dropdown_menu()
+            sel.click(self._menu_remove.format(self._div_id))
+            self.close_dropdown_menu()
 
     def minimize(self):
         """Minimize this Widget."""
         self.close_zoom()
         if not self.is_minimized:
-            sel.click(self._minimize.format(self._div_id))
```
Comment (psav) [inline cfme/dashboard.py]: DRY opportunity here?

Comment (mfalesni) [inline cfme/dashboard.py]: what exactly do you mean?

Comment (mfalesni) [inline cfme/dashboard.py]: I have changed some things for more DRY but not on this line, I don't know what would I change here.

**Comment by dajoRH** [comment]
Lint report for commit 8d359cc7d899329f0990b2ecce08efd18125cfb5:
:hurtrealbad: There were some flake issues that need to be resolved in order to merge the pull request:
```
./cfme/dashboard.py:12:1: F401 'current_version' imported but unused
```
:trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit ed4555d01327af450c3528375e0946fcdd6eb579:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 27d77c42a4c277da0d812ebdfed83400d932cd8e:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/dashboard.py (hunk 1 above)
Comment (psav) [inline cfme/dashboard.py]: I was looking at the if/else and wondering if it made sense to functionalize that and pass in the click function, but..........

Comment (mfalesni) [inline cfme/dashboard.py]: Maybe later?

**Comment by dajoRH** [comment]
Lint report for commit 86df6cd9149cee70d7d5a6e449a87b84f9b97235:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Pull request merged**